src/components/atoms/Alert/Alert.js

```diff
@@ -20,8 +20,12 @@ export default class Alert extends HTMLElement {
     alertContent.id = 'alert-content';
     this.alert.appendChild(alertContent);
 
-    shadow.addEventListener('slotchange', () => {
-      const tempElements = Array.from(this.children);
+    // TODO: See CityOfDetroit/detroitmi#1099
+    // eslint-disable-next-line no-unused-vars
+    shadow.addEventListener('slotchange', (ev) => {
+      // TODO: See CityOfDetroit/detroitmi#1099
+      // eslint-disable-next-line prefer-const
+      let tempElements = Array.from(this.children);
```

Comment:
This is from an incorrect conflict resolution from dev merge. This code shouldn't be changed in this PR. It was correct on the dev branch.

Reply:
I marked this as unresolved again because this section of code still needs to be fixed to match dev. Like this: https://is.gd/2aYdxS